xla/service/gpu/stream_attribute_annotator.cc

```diff
@@ -88,6 +91,15 @@ absl::StatusOr<bool> AnnotateStreamAttributesForInstruction(
   return true;
 }
 
+absl::StatusOr<bool> AnnotateStreamAttributesForCopyStart(
+    HloInstruction* instr, int64_t channel_id) {
+  GpuBackendConfig gpu_backend_config;
```

Comment:
Are we fine with throwing out existing `GpuBackendConfig`? What if some values are already there?

Reply:
Yes. Using GpuBackendConfig from the instruction. If some values are already there, return false to make no changes.